tests/test_game.py

```diff
@@ -47,7 +48,18 @@ def test_cycle_calls_lose_if_bet_loses(self):
     def test_choose_not_called_if_isValid_raises_exception(self):
         is_valid_mock = Mock(name="is_valid_mock", side_effect=InvalidBet)
         choose_mock = Mock(name="choose_mock")
-        with patch("roulette.Table.isValid", is_valid_mock):
+        with patch.multiple("roulette.Table", isValid=is_valid_mock), patch.multiple(
+            "roulette.Wheel", choose=choose_mock
+        ):
```

Comment:
`patch.multiple` is better suited when patching multiple functions of the same class